docs/root/operations/cli.rst

```diff
@@ -69,7 +69,8 @@ following are the command line options that Envoy supports.
 .. option:: --concurrency <integer>
 
   *(optional)* The number of :ref:`worker threads <arch_overview_threading>` to run. If not
-  specified defaults to the number of hardware threads on the machine.
+  specified defaults to the number of hardware threads on the machine. Specifically, the actual 
```

Comment:
maybe rephrase?
"If set to zero, Envoy will still run one worker thread."

Reply:
Great! Updated